docker/server/entrypoint.alpine.sh

```diff
@@ -0,0 +1,152 @@
+#!/bin/sh
+#set -x
+
+DO_CHOWN=1
+if [ "$CLICKHOUSE_DO_NOT_CHOWN" = 1 ]; then
+    DO_CHOWN=0
+fi
+
+CLICKHOUSE_UID="${CLICKHOUSE_UID:-"$(id -u clickhouse)"}"
+CLICKHOUSE_GID="${CLICKHOUSE_GID:-"$(id -g clickhouse)"}"
+
+# support --user
+if [ "$(id -u)" = "0" ]; then
+    USER=$CLICKHOUSE_UID
+    GROUP=$CLICKHOUSE_GID
+    # busybox has setuidgid & chpst buildin
```

Comment:
```suggestion
    # on alpine we use su-exec avaliable in their packages instead ot relatively heavy gosu
```

Reply:
"on alpine we use su-exec avaliable in their packages instead ot relatively heavy gosu"

It's just a comment for a reviewer. Not needed in the distribution.